src/documents/DeprecatedIn3_0.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Deprecated in 3.0
+category: Infusion
+---
+
+This page contains a list of the features, APIs, and etc. that are deprecated as of Infusion 3.0.
+
+<div class="infusion-docs-note"><strong>Note:</strong> While the intention is to provide advance notice of future
+changes, the contents of this page may not be exhaustive.</div>
+
+## Grades
```

Comment:
I realise we didn't get around to deprecating the old renderer in 2.0

We should add to this list everything in the fluid.renderer.* namespace as well as fluid.render, fluid.renderTemplates, fluid.reRender, fluid.selfRender - in fact everything in the files fluidRenderer.js, RendererUtilities.js, fluidParser.js as well as everything provided in fastXmlPull.js, that is, fluid.XMLP.* to the extent anyone was ever aware of that